app/src/main/java/fr/free/nrw/commons/upload/ReadFBMD.java

```diff
@@ -0,0 +1,47 @@
+package fr.free.nrw.commons.upload;
+
+import com.icafe4j.image.meta.Metadata;
+import com.icafe4j.image.meta.MetadataType;
+import com.icafe4j.image.meta.iptc.IPTC;
+
+import java.io.IOException;
+import java.util.Map;
+
+import javax.inject.Inject;
+import javax.inject.Singleton;
+
+import fr.free.nrw.commons.utils.ImageUtils;
+import io.reactivex.Single;
+import timber.log.Timber;
+
+@Singleton
+public class ReadFBMD {
+    public static ReadFBMD readFBMDInstance;
```

Comment:
No need to have this. As you have already annotated the class with `@Singleton` you don't need to maintain a instance manually. :)